test/createHtml.jsx

```diff
@@ -29,6 +29,50 @@ describe('createHtml()', () => {
 
   });
 
+  describe('<head/> vwo', () => {
+    it('should use vwo default account id when vwo boolean', () => {
+
+      const Html = createHtml({visualWebsiteOptimizer: true});
+      const html = $(render(<Html/>).element);
+
+      const defaultVwoAccountId = 215379;
+
+      const scriptElements = Array.prototype.slice.call(html.find('script'));
```

Comment:
Don't need to convert it to an array, the lib has some probably more readable methods. see my commit